commands/ssh.go

```diff
@@ -0,0 +1,43 @@
+package commands
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/phase2/rig/util"
+	"github.com/urfave/cli"
+)
+
+// Start is the command for creating and starting a Docker Machine and other core Outrigger services
+type Ssh struct {
+	BaseCommand
+}
+
+// Commands returns the operations supported by this command
+func (cmd *Ssh) Commands() []cli.Command {
+	return []cli.Command{
+		{
+			Name:  "ssh",
+			Usage: "Start an ssh session into the docker-machine vm",
+			Flags: []cli.Flag{
+			},
+			Before: cmd.Before,
+			Action: cmd.Run,
+		},
+	}
+}
+
+// Run executes the `rig start` command
+func (cmd *Ssh) Run(c *cli.Context) error {
+	// Does the docker-machine exist
+	if !cmd.machine.Exists() {
+		return fmt.Errorf("docker machine %s not found", cmd.machine.Name)
+	}
+
+	exitCode := util.PassthruCommand(exec.Command("docker-machine", "ssh", cmd.machine.Name))
```

Comment:
With Go you can combine this command and the boolean evaluation to have just a then dealing with error or otherwise, then just return Nil (or more specifically `return cmd.Success()`)